operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -257,8 +257,9 @@ mimir:
 runtimeConfig: {}
 
 # RBAC configuration
+# For GKE/EKS/AKS use 'type: psp'. For OpenShift use 'type: scc'
 rbac:
-  pspEnabled: true
+  type: psp
```

Comment:
Maybe PSP/SCC shouldn't be enabled by default, since PSP are already deprecated and user will run into issues with Kubernetes 1.25. As a user, I'd also expect a `create: false` option, similar to how `serviceAccount` and `ingress` is managed.

Reply:
applied suggestion @ 15c125894ee3d770e102576a30880741174c4c50